Gemfile

```diff
@@ -3,7 +3,7 @@ raise "Ruby versions less than 2.2.2 are unsupported!" if RUBY_VERSION < "2.2.2"
 # VMDB specific gems
 #
 
-gem "rails",                           "~>5.0.0.1"
+gem "rails",                           "~>5.0.0", ">= 5.0.0.1"
```

Comment:
I don't think you want this change.

the previous version was more restrictive. and the current build failure is due to lightening up that restriction